src/applications/mhv/secure-messaging/containers/Compose.jsx

```diff
@@ -69,10 +69,10 @@ const Compose = () => {
       <section className="emergency-note">
         <p>
           <strong>Note: </strong>
-          Call <a href="tel:911">911</a> if you have a medical emergency. If
-          you’re in crisis and need to talk to someone now, call the{' '}
-          <a href="tel:988">Veterans Crisis Line</a>. To speak with a VA
-          healthcare team member right away, contact your local VA call center.
+          Call <va-telephone contact="911" /> if you have a medical emergency.
+          If you’re in crisis and need to talk to someone now, call the{' '}
+          <va-telephone contact="988" />. To speak with a VA healthcare team
```

Comment:
Check this sentence... This will just render text as "988" and not as the original text you're replacing, so your sentence will read "If you're in crisis and need to talk to someone now, call the 988."

Reply:
It looks like this error was actually there previously. The secure messaging team will fix it so it reads correctly.